packages/api/src/schemas/envConfig.json

```diff
@@ -0,0 +1,54 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://github.com/OfficeDev/TeamsFx/schemas/v1.0/envConfig.json",
```

Comment:
Should this "$id" be a valid URL? like `https://raw.githubusercontent.com/OfficeDev/TeamsFx/main/packages/api/src/schemas/envConfig.json

Reply:
Removed it to avoid confusion and we can add it back when it's needed.